src/main/java/svydovets/core/context/beanFactory/command/CommandFactoryEnum.java

```diff
@@ -0,0 +1,5 @@
+package svydovets.core.context.beanFactory.command;
+
+public enum CommandFactoryEnum {
+    FC_GET_BEAN, FC_GET_BEANS_OF_TYPE;
```

Comment:
```suggestion
    FC_GET_BEAN, FC_GET_BEANS_BY_TYPE;
```

Reply:
@IgorZaporozhtsev named the same like appropriate method in BeanFactory